comm.go

```diff
@@ -5,7 +5,9 @@ import (
 	"context"
 	"io"
 
+	metrics "github.com/libp2p/go-libp2p-pubsub/metrics"
 	pb "github.com/libp2p/go-libp2p-pubsub/pb"
+	stats "go.opencensus.io/stats"
```

Comment:
please hide this behind a `metrics` interface, I don't want to depend on opencensus directly anywhere in the code except the `metrics` module.